### PR TITLE
[OCDM] Recreate RPC::CommunicatorClient connection of OCDMAccessor wh…

### PR DESCRIPTION
**Pull request by Damian-Danylko**
…en server is interrupted by crash or deactivation

Solution for WPE-252

### CODE REVIEW

File: Source/ocdm/open_cdm_impl.h [hunk 1]
```diff
@@ -47,17 +47,25 @@ struct OpenCDMAccessor : public OCDM::IAccessorOCDM,
...
-        _remote = _client->Open<OCDM::IAccessorOCDM>(_T("OpenCDMImplementation"));
-
-        printf("Trying to open an OCDM connection result %p\n", _remote);
+    void Reconnect() const
+    {
+        if (_client->IsOpen() == false) {
+            if (_remote != nullptr) {
+                _remote->Release();
+            }
+            _remote = _client->Open<OCDM::IAccessorOCDM>(_T("OpenCDMImplementation"));
```
Comment (pwielders) [inline Source/ocdm/open_cdm_impl.h]: Maybe good to clean this up while we are working on this again. This is an interface that shuld have no Ext. Move all methods on Ext to the Base interface, then remove all refernces to the Ext, and use the base.
This way there is no additional round trip needed to call the second interface.

Comment (pawel-stanek-ml) [reply Source/ocdm/open_cdm_impl.h]: +1

Comment (Damian-Danylko) [reply Source/ocdm/open_cdm_impl.h]: I did changes on Thunder/Nanoservices, let me know if it is what you are looking for.

**Comment by Damian-Danylko** [comment]
> Small remark, I am missing the removal of WPEFramework::RPC::ID_ACCESSOROCDM_EXTENSION in the ids.h file. Lets clean it all up. If you add it, it looks good to me!

Done

**Pull request merged**